matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/store/db/model/GossipingEventEntity.kt

```diff
@@ -33,6 +33,8 @@ import timber.log.Timber
  * (room key request, or sss secret sharing, as well as cancellations)
  *
  */
+
+// not used anymore, just here for db migration
```

Comment:
I think you can just remove the class, no? For the migration a String with the class name is used, so it's fine.
Same for the 2 other deleted realm classes.

Reply:
done